### PR TITLE
common: code-style related refactoring

### PR DESCRIPTION
**Pull request by naumvd95**

### CODE REVIEW

File: servers/partner-ci/9.x/nsxv.yaml [hunk 1]
```diff
@@ -527,10 +527,12 @@
...
       - shell: |
           #!/bin/bash
-          echo "TEST_GROUP=nsxv_smoke" > property
-          echo "NSXV_PLUGIN_PATH={released_plugin_path}" >> property
-          echo "PLUGIN_VERSION=3.0.0" >> property
-          echo "PKG_JOB_BUILD_NUMBER=released" >> property
+          cat <<EOF > property
+          TEST_GROUP=nsxv_smoke
+          NSXV_PLUGIN_PATH={released_plugin_path}
+          PLUGIN_VERSION=3.0.0
```
Comment (rkhozinov) [inline servers/partner-ci/9.x/nsxv.yaml]: 3.0.1 ?

**Review by rkhozinov — CHANGES_REQUESTED** [review]

File: servers/partner-ci/9.x/nsxv.yaml (hunk 1 above)
Comment (rkhozinov) [reply servers/partner-ci/9.x/nsxv.yaml]: https://review.openstack.org/#/c/401266/2/plugin_test/tests/test_plugin_nsxv.py

**Review by rkhozinov — CHANGES_REQUESTED** [review]

**Comment by rkhozinov** [comment]
need to rebase it, because master was rebased

**Comment by naumvd95** [comment]
please clarify, how can i help u

**Pull request closed without merging**